nmmo/core/env.py

```diff
@@ -260,6 +265,9 @@ def step(self, actions: Dict[int, Dict[str, Dict[str, Any]]]):
 
     rewards, infos = self._compute_rewards(self.obs.keys(), dones)
 
+    if self.config.SAVE_REPLAY:
```

Comment:
Since we always create a packet_manager, we should always call .update on it. DummyPacketManager will just drop it

Reply:
This part was moved to `Realm.step()`: `self._replay_helper.update()`

`realm.packet()` is called inside `update()` only when SAVE_REPLAY = True